app/src/main/java/org/oppia/app/help/faq/FAQActivity.kt

```diff
@@ -0,0 +1,29 @@
+package org.oppia.app.help.faq
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.app.R
+import org.oppia.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+/** The FAQ page activity for placement of different FAQs. */
+class FAQActivity : InjectableAppCompatActivity() {
+
+  @Inject
```

Comment:
This can be written in a single Line:
`@Inject lateinit var FAQActivityPresenter: FAQActivityPresenter`